Formula/pcp.rb

```diff
@@ -0,0 +1,21 @@
+class Pcp < Formula
+  desc "📦 Command-line peer-to-peer data transfer tool based on libp2p"
+  homepage "https://github.com/dennis-tra/pcp"
+  url "https://github.com/dennis-tra/pcp.git",
+      tag:      "v0.4.0",
+      revision: "7f638fe42f6dbd17e5bf5a7be5854220e2858eb2"
+  license "Apache-2.0"
+  head "https://github.com/dennis-tra/pcp.git"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-X main.RawVersion=#{version} -X main.ShortCommit=#{Utils.git_short_head(length: 7)}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "cmd/pcp/pcp.go"
+  end
+
+  test do
+    expected = "error: failed to initialize node: could not find all words in a single wordlist"
+    assert_match expected, shell_output("#{bin}/pcp receive words-that-dont-exist", 1).chomp
```

Comment:
Sorry, I should've noticed this earlier (I promise this is my last comment/suggestion here):

Can we do an `assert_equal` instead of an `assert_match` here? `assert_match` only checks that it can find `expected` as a substring in its second argument.

Reply:
No worries!

I changed it to `assert_equal`. However, I noticed that `shell_output` only captures `stdout` - the error message is printed to `stderr`, so I added the redirect via `2> &1`. If there is syntactic sugar for this, don't hesitate to point me in the right direction.

Reply:
Good catch. We really should have sugar for `2>&1` (lots of formulae do it in their tests), but alas, not yet 😅